.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
General: Consider using branch names in snake case (i.e branch_names_like_this) and add an identifier to this branch as a prefix (for example feature/add_infrastracture)